static/img/fandom_logo.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="130" height="28" viewBox="0 0 130 28" xmlns="http://www.w3.org/2000/svg">
+<svg width="130" height="28" viewBox="0 0 130 28" xmlns="https://www.w3.org/2000/svg">
```

Comment:
I dunno if this will actually break anything, but it's not really correct to use https here. See https://github.com/validator/validator/issues/692

Reply:
yeah, its a literal namespace. I dont have too much insight into it. 🤷‍♂